### DISCUSSION TITLE
Ability to use msg objects as inputs

### DISCUSSION THREAD

**Discussion by marrobHD** [zachowj/node-red-contrib-home-assistant-websocket#353] [Feature Request]
Ability to use msg objects as inputs
![image](https://user-images.githubusercontent.com/29308361/107929974-ab908300-6f7a-11eb-8e5a-63c8d3bb2ffa.png)
I marked the corresponding field.

**Comment by jacgl** [comment]
+1
currently no ability to use mustache template or msg.name
